src/corehost/common/longfile.windows.cpp

```diff
@@ -65,6 +69,19 @@ bool AssertRepeatingDirSeparator(const pal::string_t& path)
 
     assert(path_to_check.find(altDirSeparator) == pal::string_t::npos);
 
+    pal::string_t combDirSeparator1;
+    combDirSeparator1.push_back(LongFile::DirectorySeparatorChar);
+    combDirSeparator1.push_back(LongFile::AltDirectorySeparatorChar);
+
+    assert(path_to_check.find(combDirSeparator1) == pal::string_t::npos);
```

Comment:
For  example C:\users\user\..//new will work fine
But \\?\C:\users\user\..// will not work.

These asserts will prevent appending such special characters to a path. As with the extended syntax such appends will not work